public/app/features/browse-dashboards/components/CreateNewButton.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { CreateNewButton } from './CreateNewButton';
+
+async function renderAndOpen(folderUID?: string) {
+  render(<CreateNewButton inFolder={folderUID} />);
+  const newButton = screen.getByText('New');
+  await userEvent.click(newButton);
+}
```

Comment:
Normally I would discourage these types of `render-and-foo` functions in tests as they make the tests less straight forward to read (it's like learning a whole new API in every test file), and can make them more brittle to change later on.

I think there's beauty in more verbose, easier to read code, especially in tests :)

Here, I think it's fine because it's relatively straightforward.